gpbft/gpbft.go

```diff
@@ -1241,6 +1251,23 @@ func newConvergeState() *convergeState {
 	}
 }
 
+// NotifySelfConvergeBegun notifies the convergeState that the self participant
+// has begun the CONVERGE_PHASE. This notification ensures that the convergeState
+// of a round does not rely on messages broadcast by a participant destined for
+// itself to be delivered synchronously. See HasSelfBegunConverge.
+func (c *convergeState) NotifySelfConvergeBegun(value ECChain, justification *Justification) {
```

Comment:
Nit: rather than the terminology of notifications in the method names and comments, I suggest something simple and direct like `SetSelfValue`, `HasSelfValue`.